snowflake/src/integration/java/org/apache/iceberg/snowflake/NamespaceTests.java

```diff
@@ -0,0 +1,290 @@
+    }
+  }
+
+  @Test
+  public void testLoadNamespaceWithQuotedIdentifier() throws SQLException, InterruptedException {
+    String schema1 = "\"Schema123\"";
+    String dbName = TestConfigurations.getInstance().getDatabase();
+    try {
+      Assertions.assertThatNoException()
+          .isThrownBy(
+              () -> {
+                clientPool.run(conn -> conn.createStatement().execute("use " + dbName));
+                createOrReplaceSchema(schema1);
+                snowflakeCatalog.loadNamespaceMetadata(Namespace.of(dbName, schema1));
+              });
+    } finally {
+      dropSchemaIfExists(schema1);
+    }
+  }
+
+  @Test
+  public void testLoadNamespaceWithQuotedIdentifierWithSpecialCharacters()
+      throws SQLException, InterruptedException {
+    String schema1 = "\"H@!!.0-w*r()^'\"";
+    String dbName = TestConfigurations.getInstance().getDatabase();
+    try {
+      Assertions.assertThatNoException()
+          .isThrownBy(
+              () -> {
+                clientPool.run(conn -> conn.createStatement().execute("use " + dbName));
+                createOrReplaceSchema(schema1);
+                snowflakeCatalog.loadNamespaceMetadata(Namespace.of(dbName, schema1));
+              });
+    } finally {
+      dropSchemaIfExists(schema1);
+    }
+  }
+
+  @Test
+  public void testDropNamespaceNotSupported() throws SQLException, InterruptedException {
+    String schema1 = "Schema123";
+    String dbName = TestConfigurations.getInstance().getDatabase();
+    try {
+      Assertions.assertThatException()
+          .isThrownBy(
+              () -> {
+                clientPool.run(conn -> conn.createStatement().execute("use " + dbName));
+                createOrReplaceSchema(schema1);
+                snowflakeCatalog.dropNamespace(Namespace.of(dbName, schema1));
+              })
+          .isInstanceOf(UnsupportedOperationException.class);
+    } finally {
+      dropSchemaIfExists(schema1);
+    }
+  }
+
+  @Test
+  public void testSetPropertiesNotSupported() throws SQLException, InterruptedException {
+    String schema1 = "Schema123";
+    String dbName = TestConfigurations.getInstance().getDatabase();
+    try {
+      Assertions.assertThatException()
+          .isThrownBy(
+              () -> {
+                clientPool.run(conn -> conn.createStatement().execute("use " + dbName));
+                createOrReplaceSchema(schema1);
+                snowflakeCatalog.setProperties(
+                    Namespace.of(dbName, schema1), new HashMap<String, String>());
+              })
+          .isInstanceOf(UnsupportedOperationException.class);
+    } finally {
+      dropSchemaIfExists(schema1);
+    }
+  }
+
+  @Test
+  public void testRemovePropertiesNotSupported() throws SQLException, InterruptedException {
+    String schema1 = "Schema123";
+    String dbName = TestConfigurations.getInstance().getDatabase();
+    try {
+      Assertions.assertThatException()
+          .isThrownBy(
```

Comment:
`.isThrownBy(..)` should only contain the line of code that actually throws the exception. Same for all the other test methods in this class